meta-dts-distro/recipes-tests/converged-security-suite/bg-suite_2.7.0.bb

```diff
@@ -1,6 +1,6 @@
-FILESEXTRAPATHS += "${THISDIR}/${PN}"
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```

Comment:
I love that such obvious mistakes are now detected automatically by the tool.
But in this particular case, we do not need this line at all - it is required in bbappend files only.